.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Sadly, your linters are not running 😥 

This is the way your PR should display your linters checks at the end of the PR. Kindly make the necessary changes to enable the linters properly 🤓
![Captura](https://user-images.githubusercontent.com/15807343/148312128-3b6c3d07-5de3-41be-a194-f7a9a2cbee89.PNG)

Hint: I think you are missing the **_.github/workflows/linters.yml_** first step of the linters [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) 👀